js/index.js

```diff
@@ -1,3 +1,38 @@
-/* Logic for page animations and simple interactions should be placed here
-   Any page or component specific logic should be placed in its own page/component js file.
-**/
\ No newline at end of file
+let image = ['assets/img/cat1.jpg',
+             'assets/img/cat2.png', 
+             'assets/img/cat3.jpeg',
+             'assets/img/momia.jpeg'],
+    cont = 0;
```

Comment:
maybe `cont` should be named `position` or `currentPosition` to ensure it's intention is clear.